spec/controllers/posts_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+  before do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+    @post = Post.create(title: 'Title', text: 'Body', author_id: @user.id)
+  end
+
+  describe 'GET #index' do
+    it 'returns a 200 status code' do
+      subject { get :index, params: { user_id: @user.id } }
+      expect(subject).to have_http_status(200)
+    end
+    it 'render post template' do
+      subject { get :index, params: { user_id: @user.id } }
+      expect(subject).to render_template('index')
+    end
+  end
+
+  describe 'GET #show' do
+    it 'returns a 200 status code' do
+      subject { get :show, params: { user_id: @user.id, id: @post.id } }
+      expect(subject).to have_http_status(200)
+    end
+
+    it 'render show template' do
+      subject { get :show, params: { user_id: @user.id, id: @post.id } }
+      expect(subject).to render_template('show')
```

Comment:
- [ ] Following the project requirements, make sure that for each action you check:
- If response status was correct.
- If a correct template was rendered.
- If the response body includes correct placeholder text.

All your tests fail, please work on correcting them:

<img width="609" alt="image" src="https://user-images.githubusercontent.com/77190203/188280685-1cc4212e-d1a0-4fc5-8447-939b831fbf99.png">